cargo-frc/src/toolchain.rs

```diff
@@ -0,0 +1,160 @@
+#[cfg(target_os = "linux")]
+mod os {
+    use std::path::PathBuf;
+
+    pub fn install_dir() -> PathBuf {
+        dirs::home_dir()
+            .expect("Can't determine home directory")
+            .join("wpilib")
+        //.ok_or("Can't determine home directory".to_owned())
+    }
+
+    pub const TOOLCHAIN_URL_2019: &str = "https://github.com/wpilibsuite/toolchain-builder/releases/download/v2019-3/FRC-2019-Linux-Toolchain-6.3.0.tar.gz";
```

Comment:
The official install location for the toolchain changed for this year. I don't think it's worth trying to support the 2019 toolchain.

Reply:
Thanks, I missed that in the docs. I agree that it isn't necessary to support 2019, but it is useful to test multi-version support.